multinet/api.py

```diff
@@ -326,3 +328,37 @@ def delete_table(workspace: str, table: str) -> Any:
     """Delete a table."""
     db.delete_table(workspace, table)
     return table
+
+
+@bp.route("/uploads", methods=["POST"])
+def create_upload() -> str:
+    """Create a collection for multipart upload."""
+
+    sysdb = db.db("_system")
+
+    # note: arangodb collections can't start with a number
+    upload_id = f"upload_{uuid4().hex}"
```

Comment:
For workspace database names, we've been using `w-` followed by a UUID generated value. We could probably go with `u-` for these.